### PR TITLE
Add warning when size is too small

### LINKED ISSUE #86: coord_sf and silhouette size
Currently, when using `coord_sf` with a map projection (e.g. Mollweide) silhouettes can seem like they are not plotted because they are so small compared to the y-axis range vs. the size of the silhouette. We should perhaps add a simple if statement that compares the y-axis range to the `size` of silhouette and message the user if a certain threshold is exceeded, perhaps if y-axis is more than 1000 times `size`.

### PR DESCRIPTION
**Pull request by willgearty**
This throws a warning if the specified (or default) size is too small compared to the limits of the y axis (less than 1/1000th). This is mostly for when geom_phylopic is used with coord_sf and the data is projected/transformed.

I've also added an example to the vignette that showcases using coord_sf and a projection.

Fixes #86.

### CODE REVIEW

**Comment by willgearty** [comment]
I should probably adjust the vignette example to show an example using an sf transformation.

**Comment by willgearty** [comment]
This probably won't run until grImport2 is updated on CRAN.

**Comment by willgearty** [comment]
This will need to be updated for the precompiled vignettes...

**Comment by willgearty** [comment]
This is now updated for the precompiled vignettes.

**Review by LewisAJones — APPROVED** [review]
Hi @willgearty, good job, this seems to work great. Some very minor comments/suggestions!

File: vignettes/ggplot-geog-plot-3-1.png
[the remark this answers is not in the document]
Comment (willgearty) [reply vignettes/ggplot-geog-plot-3-1.png:1]: I think we should just let the PR for #87 fix that.

**Pull request merged**